en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md

```diff
@@ -14,6 +14,10 @@ The following diagram illustrates all the required functionality of the Kafka se
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
+## Set up Kafka
+
+Before you begin, set up Kafka by following the instructions in the [Setting up Kafka documentation](setting-up-kafka.md).
```

Comment:
Shouldn't this be as follows:
"Before you begin, set up Kafka by following the instructions in [Setting up Kafka](setting-up-kafka.md)"

Or,
"Before you begin, follow the instructions to [set up Kafka](setting-up-kafka.md)"

Or,
"Before you begin, [set up Kafka](setting-up-kafka.md) for the Micro Integrator"

wdyt?

Reply:
Fixed.